### ISSUE TITLE
HELLO IAM NEW HERE TRYING TO SET UP STABLE DIFFUSION AND I GET THIS WHEN I TRY TO LAUNCH IT PLEASE CAN SOMEONE HELP THANKS,  import diffusers ModuleNotFoundError: No module named 'diffusers' Press any key to continue . . .

### ISSUE THREAD

**Issue by ROB232323** [lshqqytiger/stable-diffusion-webui-directml#248]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits

### What happened?

HELLO IAM NEW HERE TRYING TO SET UP STABLE DIFFUSION AND I GET THIS WHEN I TRY TO LAUNCH IT PLEASE CAN SOMEONE HELP THANKS,  import diffusers ModuleNotFoundError: No module named 'diffusers' Press any key to continue . . . I AM TRYING TO SET IT UP ON AMD CARD I HAVE INSTALLED THE DIFFUDERS.

### Steps to reproduce the problem

1. Go to .... 
2. Press ....
3. ...
NA

### What should have happened?

NA

### Version or Commit where the problem happens

NA

### What Python version are you running on ?

Python 3.10.x

### What device are you running WebUI on?

AMD GPUs

### Cross attention optimization

Automatic

### What browsers do you use to access the UI ?

Microsoft Edge

### Command Line Arguments

```Shell
NA
```

### List of extensions

NO

### Console logs

```Shell
venv "C:\Users\Rjs Intell PC\Desktop\A1111\stable-diffusion-webui-directml\venv\Scripts\Python.exe"
fatal: No names found, cannot describe anything.
Python 3.10.6 (tags/v3.10.6:9c7b4bd, Aug  1 2022, 21:53:49) [MSC v.1932 64 bit (AMD64)]
Version: 1.5.1
Commit hash: d030f276383d0c94d69d91c5e00fc8ee84d9e01b
Launching Web UI with arguments: --xformers --autolaunch
WARNING[XFORMERS]: xFormers can't load C++/CUDA extensions. xFormers was built for:
    PyTorch 2.0.1+cu118 with CUDA 1108 (you have 2.0.0+cpu)
    Python  3.10.11 (you have 3.10.6)
  Please reinstall xformers (see https://github.com/facebookresearch/xformers#installing-xformers)
  Memory-efficient attention, SwiGLU, sparse and more won't be available.
  Set XFORMERS_MORE_DETAILS=1 for more details
Traceback (most recent call last):
  File "C:\Users\Rjs Intell PC\Desktop\A1111\stable-diffusion-webui-directml\launch.py", line 39, in <module>
    main()
  File "C:\Users\Rjs Intell PC\Desktop\A1111\stable-diffusion-webui-directml\launch.py", line 35, in main
    start()
  File "C:\Users\Rjs Intell PC\Desktop\A1111\stable-diffusion-webui-directml\modules\launch_utils.py", line 438, in start
    import webui
  File "C:\Users\Rjs Intell PC\Desktop\A1111\stable-diffusion-webui-directml\webui.py", line 54, in <module>
    from modules.call_queue import wrap_gradio_gpu_call, wrap_queued_call, queue_lock  # noqa: F401
  File "C:\Users\Rjs Intell PC\Desktop\A1111\stable-diffusion-webui-directml\modules\call_queue.py", line 6, in <module>
    from modules import shared, progress, errors
  File "C:\Users\Rjs Intell PC\Desktop\A1111\stable-diffusion-webui-directml\modules\shared.py", line 756, in <module>
    directml_do_hijack()
  File "C:\Users\Rjs Intell PC\Desktop\A1111\stable-diffusion-webui-directml\modules\dml\__init__.py", line 61, in directml_do_hijack
    import modules.dml.hijack
  File "C:\Users\Rjs Intell PC\Desktop\A1111\stable-diffusion-webui-directml\modules\dml\hijack\__init__.py", line 6, in <module>
    import modules.dml.hijack.diffusers
  File "C:\Users\Rjs Intell PC\Desktop\A1111\stable-diffusion-webui-directml\modules\dml\hijack\diffusers.py", line 2, in <module>
    import diffusers
ModuleNotFoundError: No module named 'diffusers'
Press any key to continue . . .
```

### Additional information

NA

**Comment by lshqqytiger** [comment]
May be solved in 08994e39ee1df75f8a5be28bfc8a663d1b1ac292

**Comment by traugdor** [comment]
just FYI, you aren't supposed to run `--xformers` with AMD gpus since xformers is built for CUDA which AMD does not have.
